neqo-transport/src/ecn.rs

```diff
@@ -9,18 +9,29 @@ use std::ops::{AddAssign, Deref, DerefMut, Sub};
 use enum_map::EnumMap;
 use neqo_common::{qdebug, qinfo, qwarn, IpTosEcn};
 
-use crate::{packet::PacketNumber, recovery::SentPacket};
+use crate::{
+    packet::{PacketNumber, PacketType},
+    recovery::SentPacket,
+};
 
 /// The number of packets to use for testing a path for ECN capability.
 pub const ECN_TEST_COUNT: usize = 10;
 
+/// The number of packets to use for testing the initial a path for ECN capability when exchanging
+/// Initials during the handshake. This is a lower number than `ECN_TEST_COUNT` to avoid
```

Comment:
```suggestion
/// Initials during the handshake. This is a lower number than [`ECN_TEST_COUNT`] to avoid
```

In case we ever change the constant name, `cargo doc` will fail if this is not updated as well. Neat trick to keep documentation up-to-date.